scripts/build-all.sh

```diff
@@ -22,7 +22,7 @@ echo "Runs 'sbt $TARGET' for core and examples"
 echo "========================================================================="
 
 cd $DIR/../core
-sbt --supershell=false $TARGET publishLocal
+sbt --supershell=false ";$TARGET; scalafmtCheck; publishLocal"
```

Comment:
Among the options here scalafmtCheck does not check sbt files under `project` dir while scalafmtSbtCheck does that.

Reply:
@skonto I've never seen those semicolons used before. Are you sure they are valid sbt syntax?

Reply:
Yes they are afaik: https://stackoverflow.com/questions/43341472/how-to-run-sbt-multiple-command-in-interactive-mode-as-one-command